### ISSUE TITLE
Error in installing scratchblocks by npm

### ISSUE THREAD

**Issue by dovisutu** [scratchblocks/scratchblocks#312]
When I'm installing scratchblocks (package) by npm, it shows up error.
Previously it's because `rollup` isn't installed (maybe add it in package.json), but I finally reach a place where it's `rollup` reporting error:
```
> rollup -c --environment buildTarget:PROD; exit 0;

buildTarget: PROD;
/**
 * scratchblocks v3.4.0
 * https://scratchblocks.github.io/
 * Render scratchblocks code to SVG images.
 *
 * Copyright 2013–2019, Tim Radvan
 * @license MIT
 */

exit, 0; → build/scratchblocks.min.js, build/scratchblocks.min.es.js...
[!] (node-resolve plugin) TypeError: Cannot read property 'preserveSymlinks' of undefined
TypeError: Cannot read property 'preserveSymlinks' of undefined
    at Object.buildStart (D:\Scratch-Encoder\node_modules\rollup-plugin-node-resolve\dist\rollup-plugin-node-resolve.cjs.js:246:34)
    at D:\Scratch-Encoder\node_modules\rollup\dist\rollup.js:16649:25
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:757:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3
```
and then exited. What happened and how can I install the package successfully?
PS: I'm using windows10. This may help.

**Comment by exKAZUu** [comment]
@dovisutu 
Hi, I've fixed the issue in #295, but the latest changes have still not been published on npm.
So, I'm using the following `package.json` in my projects to use the latest version. I hope it will help you.

```
  "dependencies": {
    ... snip ...
    "scratchblocks": "https://github.com/scratchblocks/scratchblocks.git",
    ... snip ...
  },
```

**Comment by dovisutu** [comment]
Thanks. It worked.
